Remove the incorrect data

pixelwake opened the pull request:
Add the resume review to the site and remove the legacy interview criteria. Add the competencies from the position descriptions.

Commit:
Remove the incorrect data

Add the resume review to the site and remove the legacy interview criteria. Add the competencies from the position descriptions.

fureigh committed:
Copy edit: lowercase for consistency

fureigh commented on pages/interview-process.md:
```diff
@@ -3,6 +3,10 @@ permalink: /interview-process/
 
+### Application review
+
+The first step of the process will be a review of the information that you have submitted as part of your application. If your application is found to be qualified, you will proceed to the interview process.
```
@rooneywp Worth saying something here about the term 'qualified' or considering using a different word? I ask because it both is loaded and has a technically specific meaning.

pixelwake replied:
@fureigh Agreed. Update the language here.

fureigh commented on pages/interview-process.md:
```diff
@@ -11,54 +15,23 @@ The first step in the interview process is the phone screen, a 30-minute phone c
-Check out more interviewing information for each of our teams. 
-
-<ul>
-	{% for interview in site.interviews %}
```
@rooneywp Can we keep these team-specific ones? I appreciate that this is trying to address #105, but the engineering rubric, for instance, includes a lot of valuable specifics that will otherwise be lost here.

fureigh committed:
Copy edits to 'qualification competencies' section

Includes hyphens to replace a mystery character.

Commit:
Reworded Application Review

Commit:
Reverting back to lowercase

Commit:
Added specific team interviews back

pixelwake commented:
@fureigh Take a look, I added the team specific back in, removed the first sentence and reworded the application section to remove qualification. I think it's ready to be merged if you do too.

fureigh committed:
Remove extra linebreak

fureigh commented on pages/interview-process.md:
```diff
@@ -3,62 +3,43 @@ permalink: /interview-process/
+{% endfor %}
 </ul>
-
-Don't see the team you're interviewing with? Design, content, and operations information will be added soon. 	
```
@rooneywp If this is still accurate, let's include it.

pixelwake replied:
@fureigh It's kind of accurate. Each position should have a defined evaluation criteria developed along side of the performance profile. How this actually gets implemented is still undefined. If we want to leave a statement like this to reduce user anxiety/mitigate the questions, I'm good with it.

fureigh replied:
@rooneywp Reducing user anxiety FTW. :) Would you prefer a more general phrasing?

fureigh commented on pages/interview-process.md:
```diff
@@ -3,62 +3,43 @@ permalink: /interview-process/
-
-After your phone interview, the 18F member you spoke to will submit their notes to our Talent Team. If we'd like to chat with you more, we'll ask you to come in to one of our office locations or join us somewhere on the internet for a couple of longer interviews. Depending on what position you're applying for, the format will vary, but generally we group interviews with multiple candidates togeth ...
-
-### Basic rubric and criteria
```
@rooneywp Has this been supplanted by the "Application qualification competencies" section? If so, let me know and I (or someone else) can add it to the engineering interviews page per #105.

pixelwake replied:
@fureigh Yes and good call on taking that over to the Engineering section.

fureigh replied:
@rooneywp :+1:

fureigh commented:
@rooneywp Thanks! Took a look and noted a couple more questions. So close to being set...

fureigh commented on pages/interview-process.md:
```diff
@@ -3,62 +3,43 @@ permalink: /interview-process/
-	{% for interview in site.interviews %}
-		<li><a href="{{site.baseurl}}{{ interview.url }}">{{ interview.title }}</a></li>
-	{% endfor %}
+{% for interview in site.interviews %}
```
@rooneywp Sometimes spacing matters, so I'm hopping in to restore it here.

fureigh committed:
Restore previous spacing to interview loop

fureigh committed:
Restoring spacing, with tabs not spaces

Once more with feeling.

Commit:
Removing last sentence of "In-person..."

I'd like to pull this whole sentence out. Technically they aren't applying for a position and I don't think that we will be following the two week sprint interview cycle, at least initially. Too many unknowns with the new process.

Depending on what position you're applying for, the format will vary, but generally we group interviews with multiple candidates together into bi-weekly hiring sprints, every other Friday.

pixelwake commented on pages/interview-process.md:
```diff
@@ -3,55 +3,39 @@ permalink: /interview-process/
-	 - Ability to translate creativity into practical solutions 
-	 - Change hearts and minds
+After your phone interview, the 18F member you spoke to will submit their notes to our Talent Team. If we'd like to chat with you more, we'll ask you to come in to one of our office locations or join us somewhere on the internet for a couple of longer interviews.
+
```
@fureigh I made the following change here:

I'd like to pull this whole sentence out. Technically they aren't applying for a position and I don't think that we will be following the two week sprint interview cycle, at least initially. Too many unknowns with the new process.

"Depending on what position you're applying for, the format will vary, but generally we group interviews with multiple candidates together into bi-weekly hiring sprints, every other Friday."

fureigh replied:
@rooneywp Okay by me. Once the new process shakes out a bit, let's revisit giving applicants a sense of the timeline to expect. I think that's adequately captured in #75.

fureigh committed:
Restore more-interviewing-specifics-coming-soon

Using the previous language, prioritizing specificity even though it's not evergreen.

fureigh commented:
@rooneywp I restored that last line about additional team-specific interview rubrics coming soon. If this all looks good to you, merge away!
